scripts/convert_lit_checkpoint.py

```diff
@@ -100,6 +100,98 @@ def copy_weights_gpt_neox(
         state_dict[to_name] = param
 
 
+def copy_weights_llama(
```

Comment:
The function is not used anywhere. We need to wire it up in the

```py
 elif config._mlp_class == "LLaMAMLP":
        raise NotImplementedError(f"Conversion for {model_name} is not yet supported")
```

branch below in `convert_lit_checkpoint`, right?

Reply:
updated in [ef64e6c](https://github.com/Lightning-AI/lit-gpt/pull/315/commits/ef64e6c2bf47aa3b1c99d11ef2f3d230a5b506e3)